static/app/components/events/interfaces/frame/lineV2/expander.tsx

```diff
@@ -17,21 +15,14 @@ type Props = {
   isHoverPreviewed?: boolean;
 };
 
-function Expander({
-  isExpandable,
-  isHoverPreviewed,
-  isExpanded,
-  platform,
-  onToggleContext,
-}: Props) {
+function Expander({isExpandable, isHoverPreviewed, isExpanded, onToggleContext}: Props) {
   if (!isExpandable) {
     return null;
   }
 
   return (
     <StyledButton
       className="btn-toggle"
-      css={isDotnet(platform) && {display: 'block !important'}} // remove important once we get rid of css files
```

Comment:
I think this might still be necessary, otherwise the toggle button won't get displayed when it should be:

![CleanShot 2023-03-28 at 17 04 34](https://user-images.githubusercontent.com/10888943/228393499-2cfd13d2-e9f6-4f62-b574-81e0754039f0.png)

vs what it was before:

![CleanShot 2023-03-28 at 17 06 31](https://user-images.githubusercontent.com/10888943/228393611-348210d7-2de4-4a06-9158-13291153ae47.png)

This weirdness is due to do this line in the old less styles: https://github.com/getsentry/sentry/blob/4b0b43093ad1c399ddafe72c89ed3cde6f6d0d63/static/less/group-detail.less#L575

TBH I think you might be able to just delete that line since the toggle button shouldn't even be rendered when it is not expandable.

Reply:
Good catch!  I will pushed a change to remove that line.  It had the desired effect in my dev environment.  Lets see what else it affects in the visual snapshots.

Reply:
It did indeed change a couple of other screenshots.  in all cases, showing the toggle button in the collapsed state where it was previously hidden.  However, it seems like in all those cases the toggle button _should_ be visible - because there's additional information to be seen if you click to expand the frame.

Just as a sanity check, using sentry.io, I created a new JavaScript Browser project and clicked the "add a sample event" on the issues page.  Indeed, the event it created looks like it's missing the toggle button.

<img width="821" alt="image" src="https://user-images.githubusercontent.com/1396388/228422400-f77cab1b-cae6-461a-99cf-0bab461065c6.png">

Expand that first frame and the toggle button shows up.

<img width="815" alt="image" src="https://user-images.githubusercontent.com/1396388/228422493-945e3935-d176-45e6-b113-22bcaf4c35cd.png">


I then reproduced this in my dev environment.  With the less line in place, it looks the same as above.  With it removed, the toggle but is now present.

<img width="1126" alt="image" src="https://user-images.githubusercontent.com/1396388/228422681-7417f41f-ac69-4791-9abe-1f767711f170.png">

So this is a good change, yes?

Reply:
Ultimately, wasn't the problem that `isDotnet` was too narrow of a check for this CSS change?  There are too many other reasons why you might want to show a frame in a collapsed state.  It just happened that .NET always has the Assembly Info section, but anything that would be in that section (such as source context, etc.) would also be a reason to have the toggle button.

Reply:
That's awesome! I think you're right, this was probably an existing bug. These stack trace styles can get a bit hairy with that old css hanging around.

Reply:
good catch you both 🎉